lib/flow_commerce/flow_api_v0_client.rb

```diff
@@ -23086,10 +23145,10 @@ def do_request(klass)
               request.add_field(key, value)
             }
 
-            curl << "'%s'" % uri
+            curl << "'%s%s'" % [@base_uri, path]
```

Comment:
we could - but I think would prefer not to make full_uri a static method (and otherwise we can't guarantee that the provided http handler implements full_uri) - so prefer to leave as is